test/setup.ts

```diff
@@ -0,0 +1,72 @@
+import { ApolloClient, from, HttpLink, InMemoryCache } from "@apollo/client";
+import { onError } from "@apollo/client/link/error";
+import { setupPolly } from "setup-polly-jest";
+import { Polly } from "@pollyjs/core";
+import fetch from "cross-fetch";
+import NodeHttpAdapter from "@pollyjs/adapter-node-http";
+import FSPersister from "@pollyjs/persister-fs";
+import path from "path";
+
+import { SaleorSDK } from "../src/core";
+import { API_URI } from "../config";
+
+Polly.register(NodeHttpAdapter);
+Polly.register(FSPersister);
+
+export const setupRecording = () =>
+  setupPolly({
+    adapterOptions: {
+      fetch: {
+        context: global,
+      },
+    },
+    adapters: ["node-http"],
+    matchRequestsBy: {
+      headers: {
+        exclude: ["authorization"],
```

Comment:
Wouldn't `set-cookie` work here instead of filtering it out manually?

Reply:
Unfortunately, no. `matchRequestsBy` modifies only the fields specified in docs for `recording.request`